client/app/scripts/actions/app-actions.js

```diff
@@ -580,16 +580,17 @@ export function receiveControlNodeRemoved(nodeId) {
   };
 }
 
-export function receiveControlPipeFromParams(pipeId, rawTty) {
+export function receiveControlPipeFromParams(pipeId, rawTty, resizeTtyControl) {
```

Comment:
I'd try to avoid to have the signature become too TTY heavy. Pipes should be general. Maybe condense to a parameter field?